resolvers.js

```diff
@@ -206,7 +206,11 @@ const resolvers = {
   },
 
   Model: {
-    category: parent => categoryService.getCategoryById(parent.category)
+    category: parent => categoryService.getCategoryById(parent.category),
+    sizes: parent =>
+      parent.sizes.map((size, index) =>
+        sizeService.getSizeById(parent.sizes[index])
+      ),
```

Comment:
why you use as parameter `parent.sizes[index]`. You can just use `size` from your `map` arguments

Reply:
done